examples/allocation/zig/README.md

```diff
@@ -22,8 +22,3 @@ This example uses `@panic()` rather than `unreachable` to handle errors
 since `unreachable` emits a call to panic only in `Debug` and `ReleaseSafe`
 mode. In `ReleaseFast` and `ReleaseSmall` mode, it would lead into undefined
 behavior.
-
-If building wasm with a pre-release version of Zig 0.10.0, use `-fstage1` to
-avoid [bugs in the new compiler][1].
-
-[1]: https://github.com/ziglang/zig/wiki/Self-Hosted-Compiler-Upgrade-Guide#is-it-time-to-upgrade
```

Comment:
since the bootstrapped compiler is now the default and 0.10.1 has been released I am assuming this note is redundant now?